sirepo/simulation_db.py

```diff
@@ -535,6 +535,25 @@ def process_simulation_list(res, path, data):
     ))
 
 
+def read_data_file(filepath):
```

Comment:
This doesn't belong here. and use `mimetypes.guess_type(f.basename) or application/octect`

Reply:
Ok, where should I put it? In job_api?

Reply:
Yes, job_api. It's very specific.

I think we need to rename JOB_FILE (for sim.lib_file) and FILE to LIB_FILE_URI and DATA_FILE_URI. Everything associated with DATA_FILE should be data_file, DataFile, DATA_FILE, or dataFile, not just FIleReq or file. There are two different paths for different problems so they need to be distinguished clearly.

Reply:
Agreed. Renamed to DATA_FILE